### DISCUSSION TITLE
Almost there

### DISCUSSION THREAD

**Discussion by oneSALTycracker** [ImRidingNow/PavlovRconWebserver#50] [General]
the way the quest is when you tab out of the menu it will make u close it but with a saved password its not to long

 its quiet impressive how well and responsive  it is 
 
 very well done

Pictures hosted on our cloud 
https://cloud.jtwp.org/s/N9jzQRqtrQPTnnH
https://cloud.jtwp.org/s/nGfodmWeKJEeHZP
https://cloud.jtwp.org/s/ZpfiWtk3pJ7sRdj
https://cloud.jtwp.org/s/LdAWbWdRZf3LTYe

**Comment by ImRidingNow** [comment]
Oh cool, thanks, I've never seen that on the quest :D (cause i don't have one xD)

It's a bit of a shame that the quest closes the browser all the time :(

1. Does anyone else know a workaround that may not include saving the password?
2. Are you on the newest Quest update? I thought the Quest can handle this now xD
